tests/utils/utils.go

```diff
@@ -55,3 +56,12 @@ func ReadHexTextFile(filename string) string {
 	Expect(err).Should(BeNil())
 	return string(fileData)
 }
+
+// HashSliceToBytes serializes a []common.Hash into a tightly packed byte array.
+func HashSliceToBytes(hashes []common.Hash) []byte {
```

Comment:
do we need this? We can use subnet-evms

Reply:
Fixed. Previously, I thought it was not provided in the 0.5.8.